lib/std/os/linux/seccomp.zig

```diff
@@ -6,16 +6,14 @@
 //! isn't that useful for general-purpose applications, and so a mode that
 //! utilizes user-supplied filters mode was added.
 //!
-//! Seccomp filters are classic BPF programs, which means that all the
-//! information under `std.x.net.bpf` applies here as well. Conceptually, a
```

Comment:
This documentation doesn't make sense since it references quite a bit from the deleted `std.x.net.bpf`. As the author, I'd like it to be moved under `std.net`, least until the package manager arrives and the stdlib gets thinned out.

In #10717, @matu3ba wanted to create a system to run tests in child processes that communicate their status via sockets. Not sure if this exists now.

Reply:
> that communicate their status via sockets. Not sure if this exists now.

No, this would have introduced either the port rebinding problem with unnecessary+annoying waiting time or global state+handling to the system. I have an incomplete version using std.x.net here though: https://github.com/matu3ba/zig/commits/test_panics.

My idea for the test system was to make things for nonstandard pipes more declarative, but I need to change things to support the use case of writing pipe ids on stdin or doing arbitrary operations like logging.
The annoying part is that this introduces another syscall to unset the cloexec flag before the fork or more potential time to leak the pipe.
If you are interested, have a look at https://github.com/ziglang/zig/pull/12754.